packages/schematics/migrations/update-6-1-1/files/karma.conf.ts__tmpl__

```diff
@@ -0,0 +1,33 @@
+// Karma configuration file, see link for more information
+// https://karma-runner.github.io/1.0/config/configuration-file.html
+
+import { join } from 'path';
```

Comment:
Why is a ts file instead of a js file?

There is no tsconfig that we can use to compile. The root one should only be used for editor support.